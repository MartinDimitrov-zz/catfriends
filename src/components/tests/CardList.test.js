import React from 'react';
import { shallow } from 'enzyme';
import CardList from '../CardList';

it('expect to render Card component', () => {
    const mockCats = [
        {
            id: 1,
            name: 'John Smith',
            username: 'johny',
            email: 'some@email.com'
        }
    ];
    
    expect(shallow(<CardList cats={mockCats}/>).debug()).toMatchSnapshot();
});