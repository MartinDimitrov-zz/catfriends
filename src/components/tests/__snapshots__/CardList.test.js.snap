// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card component 1`] = `
"<div>
  <Card id={1} name=\\"John Smith\\" email=\\"some@email.com\\" />
</div>"
`;
